GameData/RealismOverhaul/RO_SuggestedMods/SOCK/RO_ShuttleOV.cfg

```diff
@@ -0,0 +1,622 @@
+// Shuttle Orbiter Construction Kit
+
+// Orbiter configs
+@PART[benjee10_shuttle_forwardFuselage]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.81
+	@mass = 9.16
+	@maxTemp = 1400
+	%skinMaxTemp = 2700
+	%heatConductivity = 0.01
+	%skinSkinConductionMult = 0.1
+
+	!RESOURCE[MonoPropellant]{}
+	!MODULE[ModuleReactionWheel]{}
+
+	@MODULE[ModuleCommand]
+	{
+		RESOURCE
+		{
+			// https://science.ksc.nasa.gov/shuttle/technology/sts-newsref/sts-eps.html
+			name = ElectricCharge
+			rate = 14 //Average power draw of orbiter
+		}
+	}
+
+	// https://science.ksc.nasa.gov/shuttle/technology/sts-newsref/sts-rcs.html
+	// https://spaceflight.nasa.gov/shuttle/reference/shutref/orbiter/rcs/overview.html
+	// https://www.alternatewars.com/BBOW/Space_Engines/Shuttle_Propulsion_1990.pdf
+	// https://www.smashwords.com/extreader/read/126031/3/americas-space-shuttle-reaction-control-system-nasa-astronaut-training-manual-rcs-2102a
+	// 418.6 kg MMH (475.76 L) and 664 kg (461.1 L) NTO RCS propellant
+	// RCS ISP taken from https://www.researchgate.net/profile/Btc-Zandbergen/publication/267047129_Modern_Liquid_Propellant_Rocket_Engines_2000_Outlook/links/5444b3dc0cf2a76a3ccd7b48/Modern-Liquid-Propellant-Rocket-Engines-2000-Outlook.pdf
+	// Primary RCS thrust : 3.87 kN (870 lbf)
+	
+	@MODULE[ModuleRCSFX],*
+	{
+		!resourceName = DELETE
+		@thrusterPower = 3.87
+		PROPELLANT
+		{
+			name = MMH
+			ratio = 0.4943
+		}
+		PROPELLANT
+		{
+			name = MON3
+			ratio = 0.5057
+		}
+		PROPELLANT
+		{
+			name = Helium
+			ratio = 12.9
+			ignoreForIsp = True
+		}
+		@atmosphereCurve
+		{
+			@key,0 = 0 306
+			@key,1 = 1 110
+		}
+	}
+
+	// https://www.bernd-leitenberger.de/space-shuttle.shtml
+	// three nickel-cadmium batteries with 10 amperehours each at 28 volts, so in total 3024 kJ
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 4000
+		type = ServiceModule
+		basemass = -1
+		TANK
+		{
+			name = MMH
+			amount = 475.76
+			maxAmount = 475.76
+		}
+		TANK
+		{
+			name = MON3
+			amount = 461.1
+			maxAmount = 461.1
+		}
+		TANK
+		{
+			name = Helium
+			amount = 12086
+			maxAmount = 12086
+		}
+		TANK
+		{
+			name = ElectricCharge
+			amount = 3024
+			maxAmount = 3024
+		}
+	}
+}
+
+// Cockpit internal configuration
+@INTERNAL[OV_shuttleIVA]:FOR[RealismOverhaul]
+{
+	%scaleAll = 1.78, 1.78, 1.78
+
+	@MODULE[InternalSeat],*
+	{
+		%kerbalScale = 1.78, 1.78, 1.78
+		%kerbalOffset = 0.0, 0.0, 0.0
+		%kerbalEyeOffset = 0.0, 0.0, 0.0
+	}
+}
+
+@PART[benjee10_shuttle_midFuselage]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.81
+	@mass = 18.81
+	@maxTemp = 2200 // High value to prevent payload from melting
+	%skinMaxTemp = 2400
+	%heatConductivity = 0.01
+	%skinSkinConductionMult = 0.1
+
+	%rescaleCube = 2
```

Comment:
Considering that `rescaleCube` is a marker, putting a `2` instead of, ex. `true`, could be misleading for the reader.